SPECS/calamares/calamares.spec

```diff
@@ -41,34 +40,32 @@ Source42:       calamares-auto_de.ts
 # lupdate-qt5 show.qml -ts calamares-auto_it.ts
 # then translate the template in linguist-qt5.
 Source43:       calamares-auto_it.ts
-Source52:       mariner-welcome.png
-Source53:       mariner-eula
+Source52:       azl-welcome.png
+Source53:       azl-eula
 # adjust some default settings (default shipped .conf files)
-Patch0:         calamares-3.2.11-default-settings.patch
-Patch1:         use-single-job-for-progress-bar-value.patch
-Patch2:         navigation-buttons-autodefault.patch
-Patch3:         round-to-full-disk-size.patch
+Patch0:         Azure-Linux-Calamares-Conf-Patch-3.3.1.patch
+#Patch3:         round-to-full-disk-size.patch
```

Comment:
These patches were commented out.  Do we still need them?

Reply:
The commented out patches are some that I am not currently sure if they will be needed once we are fully able to run calamares in the ISO. Instead of removing them entirely from the spec, I simply commented them out temporarily. If they are needed however, I'll like to just add them directly to the module sources instead of in a patch like this.

Reply:
ack